src/forms/EditListingSubjectForm/CustomClassLevelSelectFieldMaybe.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
is this file similar to `CustomPaymentTypeSelectFieldMaybe.js` ? Can we make it only one component for both of them?